VerifyReplication/verifyReplication.js

```diff
@@ -41,26 +42,20 @@ function verifyObjects(objectList, cb) {
 
         return destinationStorage.getObjMd(params, (err, dstMd) => {
             ++statusObj.dstProcessedCount;
-            if (err && err.code !== 'NotFound') {
+            if (err) {
                 ++statusObj.dstFailedMdRetrievalsCount;
```

Comment:
should not be increased if `notfound`:

```suggestion
if (err instanceof NotFound) {
    ++statusObj.missingInDstCount;
} else {
    ++statusObj.dstFailedMdRetrievalsCount;
}
```